jgiven-examples/src/test/java/com/tngtech/jgiven/examples/webdriver/WebDriver.java

```diff
@@ -0,0 +1,10 @@
+package com.tngtech.jgiven.examples.webdriver;
+
+public class WebDriver {
```

Comment:
maybe you could add the missing dependency to the build.gradle instead? Would also result in the correct package then:

org.seleniumhq.selenium:htmlunit-driver:2.21